assets/create-pad.css

```diff
@@ -1,5 +1,5 @@
 .djs-palette {
-  z-index: 200;
+  z-index: 400;
```

Comment:
We have to do this because the palette doesn't have any z-index by default. Alternatively, we could add one in diagram-js but it's not strictly needed there.

Reply:
:+1: